Cargo.toml

```diff
@@ -51,6 +50,12 @@ sha2 = "0.6"
 abxml = "0.2"
 handlebars = "0.29"
 
+[target.'cfg(windows)'.dependencies]
+colored = { version: "^1.4", features: ["no-color"] }
```

Comment:
This is making builds fail. It should be:

```toml
colored = { version = "^1.4", features = ["no-color"] }
```

Reply:
Whoops, confused TOML with JSON there. :roll_eyes: